AllReadyApp/Web-App/AllReady/Views/Manage/Index.cshtml

```diff
@@ -202,7 +202,8 @@
             </div>
             <div class="form-group">
                 <div class="col-md-offset-2 col-md-10">
-                    <button type="submit" class="btn btn-default" data-bind="enable: skills.isValid">Save</button>
+                    <button type="submit" name="button" value="Save" class="btn btn-default" data-bind="enable: skills.isValid">Save</button>
+                    <button type="submit" name="button" value="Cancel" class="btn">Cancel</button>

```

Comment:
If we make this a link back to the index page (instead of a button), we won't need to add any special logic to the controller.